sdk/bulk.go

```diff
@@ -82,9 +95,45 @@ func (s SDK) BulkDecrypt(ctx context.Context, opts ...BulkDecryptOption) error {
 	tdfDecryptors := make(map[string]decryptor)
 	policyTDF := make(map[string]*BulkTDF)
 
+	switch bulkReq.TDFType {
+	case Nano:
+		dummy := &NanoTDFReaderConfig{}
+		for _, opt := range bulkReq.NanoTDFDecryptOptions {
+			_ = opt(dummy)
+		}
+		if !dummy.ignoreAllowList && dummy.kasAllowlist == nil {
```

Comment:
Instead of checking `nil` it may be more robust to instead check `len(..) == 0`, that way empty slices are also handled.

Reply:
Today I Learned: https://go.dev/play/p/3sq57Tmcy7C